aperture.go

```diff
@@ -138,6 +152,17 @@ func start() error {
 			}
 			log.Infof("Done generating TLS certificates")
 		}
+
+		// Load the certs now so we can create a complete TLS config.
+		certData, _, err := cert.LoadCert(tlsCertFile, tlsKeyFile)
```

Comment:
Cert loading needs to move to the next commit?

Reply:
Not really, we can't create a valid `TLSConfig` without the certificate.